### DISCUSSION TITLE
I cant fined the integration, is it just me?

### DISCUSSION THREAD

**Discussion by salibaba1971** [maorcc/citymind_water_meter#40] [Q&A]
I am on Home Assistant 2022.11.2
Supervisor 2022.10.2
Operating System 9.3
Frontend 20221108.0 - latest
I cant fined City-Mind Water Meter at the general search of HACS and also it is not at Settings->Devices & Services->add integration
Do I do something wrong?

![1](https://user-images.githubusercontent.com/117904816/201067813-a0bb2874-bab2-431e-9e46-bc1212afd834.jpg)
![2](https://user-images.githubusercontent.com/117904816/201067835-0576fa9c-ecf1-430b-b5c3-26c1fe50124d.jpg)

**Comment by elad-bar** [comment]
This screenshot is for integrations, not hacs

**Reply by salibaba1971** [reply]
Thanks, 
I thought HASC is the home assistant.
I installed the HASC and added the integration. It works perfectly.
